bundles/ch.elexis.mednet.webapi.ui/src/ch/elexis/mednet/webapi/ui/preferences/MedNetWebPreferencePage.java

```diff
@@ -25,47 +22,38 @@
 @Component
 public class MedNetWebPreferencePage extends FieldEditorPreferencePage implements IWorkbenchPreferencePage {
 
-    private IConfigService configService;
+	private IConfigService configService;
```

Comment:
Bitte nicht die `PreferencePage` mit `@Component` als OSGi Service registrieren.

Sondern das `IConfigService` Feld mit `@Inject` annotieren, und im Konstruktor der `PreferencePage` mit `CoreUiUtil.injectServices(this);` den Service holen.